### PR TITLE
AS4 Upgrade 

### PR DESCRIPTION
**Pull request by mabuyo**
Upgrades to use Apollo Server 4.

DO NOT MERGE until course content is also updated.

### CODE REVIEW

**Review by lizhennessy — APPROVED** [review]
✅ starter code works great
✅ final folder code works great

File: final/server/src/schema.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { gql } = require('apollo-server');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/server/src/schema.js:1]:
```suggestion
const gql = require('graphql-tag');
```

File: server/src/schema.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { gql } = require('apollo-server');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline server/src/schema.js:1]:
```suggestion
const gql = require('graphql-tag');
```

File: final/server/src/schema.js (hunk 1 above)
Comment (mabuyo) [inline final/server/src/schema.js:1]:
```suggestion
const gql = require('graphql-tag');
```

**Pull request merged**